react/MuiCozyTheme/Menus/index.js

```diff
@@ -1,3 +1,121 @@
-import Menu from '@material-ui/core/Menu'
+import React, { Component } from 'react'
 
-export default Menu
+import MenuButton from '../../Button'
+import PropTypes from 'prop-types'
+
+import ClickAwayListener from '@material-ui/core/ClickAwayListener'
+import Grow from '@material-ui/core/Grow'
+import Popper from '@material-ui/core/Popper'
+import Paper from '@material-ui/core/Paper'
+import MenuList from '@material-ui/core/MenuList'
+import { withStyles } from '@material-ui/core/styles'
+
+const styles = {
+  paper: {
+    '& hr': {
+      margin: '8px 0',
+      border: 0,
+      borderTop: '1px solid #D6D8DA'
+    }
+  }
+}
+
+class Menu extends Component {
+  state = { open: false, anchorEl: null }
+
+  toggle = event => {
+    const { currentTarget } = event
+    this.setState(state => ({
+      anchorEl: currentTarget,
+      open: !state.open
+    }))
+  }
+
+  close = () => {
+    this.setState({
+      open: false
+    })
+  }
+
+  render() {
+    const {
+      label,
+      disabled,
+      buttonClassName,
+      component,
+      children,
+      placement,
+      classes
+    } = this.props
+    const { open, anchorEl } = this.state
+
+    return (
+      <ClickAwayListener onClickAway={this.close}>
+        <div>
+          {!component ? (
+            <MenuButton
+              disabled={disabled}
+              onClick={this.toggle}
+              label={label}
+              buttonClassName={buttonClassName}
+            />
+          ) : (
+            React.cloneElement(component, {
+              disabled,
+              onClick: this.toggle
+            })
+          )}
+          {
+            <Popper
+              open={open}
+              anchorEl={anchorEl}
+              transition={open}
```

Comment:
I think this is why the transition does not work when closing. Since `transition={false}` in this case, transitions are not activated and Popper cannot support a Transition component like `Grow`.

> transition: Help supporting a react-transition-group/Transition component.